### PR TITLE
Automation Update

### PR DESCRIPTION
**Pull request by MyDrift-user**
## Type of Change
- [x] Refactoring

## Description
Improve Automation:
- Instead of checking checkboxes and run tweaks run tweaks using config (remove UI dependency)
- Don't generate UI on automation (improves speed)

## TODO
- [ ] run automation in synchronogical order (on automation)

## Impact
Trying to massively improve automation without having to change the user's existing configuration.

## Checklist
- [ ] My code adheres to the coding and style guidelines of the project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: scripts/main.ps1 [hunk 1]
```diff
@@ -173,12 +175,62 @@ Invoke-WPFRunspace -ScriptBlock {
...
+                # Create a new object with the combined values
+                $combinedEntry = @{
+                    Name = $wpfInstallEntry
+                    Winget = $installEntry.winget
+                    Choco = $installEntry.choco
+                }
+
+                # Add the combined entry to the array
+                $combinedInstallConfig += $combinedEntry
+            }
```
Comment (momcilovicluka) [inline scripts/main.ps1:210]: Hi! 😊 Great work as always! I suggest using a foreach loop with the range operator for combining the installation configs to make the code shorter
```suggestion
            $combinedInstallConfig = foreach ($i in 0..($installConfig.WPFInstall.Count - 1)) {
                @{
                    Name  = $installConfig.WPFInstall[$i]
                    Winget = $installConfig.Install[$i].winget
                    Choco  = $installConfig.Install[$i].choco
                }
            }
```

Comment (MyDrift-user) [reply scripts/main.ps1]: oh thanks, way better this way :)

Comment (momcilovicluka) [reply scripts/main.ps1]: You're welcome! I'm glad you liked it!

**Comment by MyDrift-user** [comment]
closing this out due to inactive work. Just have to much to do rn, will revisit this as soon as possible :)

**Pull request closed without merging**